src/current/_includes/releases/v23.1/v23.1.3.md

```diff
@@ -0,0 +1,147 @@
+## v23.1.3
+
+Release Date: June 12, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-3-security-updates">Security updates</h3>
+
+- Added a new `server.client_cert_expiration_cache.capacity` [cluster setting](../v23.1/cluster-settings.html), which when set to a non-zero number will store the minimum time-until-expiration for the set of client certificates per user. This setting can be used to export client certificate expirations as a metric. For example, if set to zero, the `security.certificate.expiration.client` metric will have a value of zero. [#104165][#104165]
+
+<h3 id="v23-1-2-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Role-based audit logging is now an {{ site.data.products.enterprise }} feature. Only {{ site.data.products.enterprise }} users will be able to configure role-based audit logging using the `sql.log.user_audit` [cluster setting](../v23.1/cluster-settings.html). [#104453][#104453]
+- Introduced role-based auditing through the new `sql.log.user_audit` [cluster setting](../v23.1/cluster-settings.html). Using the cluster setting, a user is able to set an audit logging configuration using a table-like syntax. Each row in the configuration represents an _audit setting_ in the configuration. An audit setting is comprised of the following columns: `USER/ROLE` and `STATEMENT_FILTER`:
```

Comment:
This release note I have tried to summarize from the longer original. There aren't yet docs on this, so maybe we should include the full original draft (edited).

Reply:
Makes sense.